processes/security_team_members.md

```diff
@@ -0,0 +1,92 @@
+# Node.js Security Team
+
```

Comment:
I am assuming this is intended specifically to describe the security team for Node.js *core* rather than the third party modules process, correct? If so, it would be good to state that up front.

Reply:
It is will eventually describe all the Node.js Security Teams, as the same privacy policy will apply to all of them.

The third party module process and teams are not yet listed, because they don't exist (yet).